deps/jerry/api.cc

```diff
@@ -1965,8 +1965,19 @@ MaybeLocal<Value> v8::Object::GetOwnPropertyDescriptor(Local<Context> context,
   V8_CALL_TRACE();
   JerryValue* jobject = reinterpret_cast<JerryValue*>(this);
   JerryValue* jkey = reinterpret_cast<JerryValue*>(*key);
+  JerryValue* property_object;
 
-  JerryValue* property_object = jobject->GetOwnPropertyDescriptor(*jkey);
+  jerry_property_descriptor_t prop_desc;
+  bool has_own_property = jerry_get_own_property_descriptor (jobject->value(), jkey->value(), &prop_desc);
+
+  if (has_own_property)  {
+      property_object = new JerryValue (jerry_from_property_descriptor (&prop_desc));
+      jerry_free_property_descriptor_fields(&prop_desc);
+      RETURN_HANDLE(Value, context->GetIsolate(), property_object);
+  }
+
+  property_object = new JerryValue ();
+  jerry_free_property_descriptor_fields(&prop_desc);
   RETURN_HANDLE(Value, context->GetIsolate(), property_object);
```

Comment:
If the property does not exist you should return an empty handle.
`return MaybeLocal<Value>()`